src/components/organisms/Badges/BadgeImage.tsx

```diff
@@ -18,8 +18,8 @@ export const BadgeImage: React.FC<{
     <img
       className="badge-list-item-image"
       ref={imageRef}
-      src={image ?? defaultAvatar}
-      alt={`${label} badge`}
+      src={image}
+      alt={""}
```

Comment:
Why are we removing the label from here?

Reply:
Because the image is for decoration only and if the URL is invalid this alt text shows up for a split second until onError is called which defaults to the default avatar.